### PR TITLE
Create CODEOWNERS: adding code owners file to make it work, approvals

### PR DESCRIPTION
**Pull request by AlxdrPolyakov**
## Context

Create CODEOWNERS: adding code owners file to make it work, approvals
Create CODEOWNERS: adding code owners file to make it work, approvals
Create CODEOWNERS: adding code owners file to make it work, approvals
Create CODEOWNERS: adding code owners file to make it work, approvals
Create CODEOWNERS: adding code owners file to make it work, approvals

## Checklist
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

**Review by moulougeta — APPROVED** [review]

**Review by AlxdrPolyakov — COMMENTED** [review]
Approved

File: .github/CODEOWNERS
Comment (TDebonoWise) [inline .github/CODEOWNERS:1]: /wise-bot approve

**Pull request merged**